.github/workflows/rust-audit.yml

```diff
@@ -0,0 +1,27 @@
+name: Rust Audit for Security Vulnerabilities
```

Comment:
Crontab based run of Rust Audit: every weekday, at 11:00 (just before the Daily). If jobs fails, a message is posted on Slack.

Reply:
Nice!